keylime/tenant.py

```diff
@@ -642,66 +644,77 @@ def do_status(self, listing=False, returnresponse=False, bulk=False):
                 cert=self.cert,
                 verify=False
             )
+            response = response.json()
         else:
             response = do_cvstatus.get(
                 (f'/v{self.api_version}/agents/{agent_uuid}'),
                 cert=self.cert,
                 verify=False
             )
+            response = response.json()
```

Comment:
This part should happen after we check the return codes. It's ok to create the object we will eventually return here and then augment it in the following section.